apps/admin/vite.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
   },
   test: {
     coverage: {
-      enabled: true,
+      provider: 'istanbul',
```

Comment:
Why did you change this?

Reply:
If this file is checked with linting with type checking it will throw an error since `enabled:true` is not an option. Therefore I set it to istanbul. This only occurs when you remove the file from the include list here: apps/admin/tsconfig.json. It has something to do with type-aware linting, but I don't fully understand how it works. 

![Screenshot 2025-01-21 at 08 53 21](https://github.com/user-attachments/assets/0809cb15-4aaa-455a-bc0f-2803f7ff43a6)

Reply:
It's not an issue when the file is in the `include` list, right? The `vite/client` types are set in that `tsconfig` as well, so it makes sense it needs to be included I think. 

When `vite.config.ts` is included in `tsconfig`, `enabled: true` seems to work, so I wouldn't change it.

Reply:
Alright, I was under the impression it gave also a more visual presentation of the coverage. But that is not the case. I'll move it back.